engine/flink/components/table/src/main/scala/pl/touk/nussknacker/engine/flink/table/utils/DataTypeFactoryHolder.scala

```diff
@@ -0,0 +1,46 @@
+package pl.touk.nussknacker.engine.flink.table.utils
```

Comment:
I would move all this `SimulatedDataTypeFactory`, ...Converter and CallContext to one package - `simulateddatatype`